### PR TITLE
Add additional spaces

### PR DESCRIPTION
**Pull request by dahvreinhart**
- Found new button images and got ready for buttons
- Added new space buttons
- Enabled a handful of spaces
- Took out empty boxes
- Fixed chat window title alignment

### CODE REVIEW

**Comment by HaltonO** [comment]
Love the pictures lol 👍

**Comment by HaltonO** [comment]
the footer is flying.. you can just put this at the end of the page
 <div className="empty-box" />

and in css:
.empty-box{
  height:500px;
}

File: imports/ui/GridView/index.js [hunk 1]
```diff
@@ -10,16 +10,38 @@ export default class GridView extends Component {
     	return(
```
Comment (madeline-petersen) [inline imports/ui/GridView/index.js]: Do you have "Fix errors on save" checked? My editor corrected all the indentation automatically.
Atom -> Preferences -> Packages -> linter-eslint -> 
- [x] Fix errors on save

**Comment by madeline-petersen** [comment]
I experimented with Flexbox a little to make the grid centered... by using their flex-wrap property we can have our grid adjust based on the size of the screen. What do you think?  It fit four across, but we could make the column narrower to make three across the maximum.

screen shot 2016-11-30 at 8 59 05 pm
screen shot 2016-11-30 at 8 58 47 pm
screen shot 2016-11-30 at 8 58 37 pm
screen shot 2016-11-30 at 8 58 21 pm

**Pull request merged**